airflow/providers/google/ads/hooks/ads.py

```diff
@@ -31,9 +31,9 @@
 from airflow.providers.google.common.hooks.base_google import get_field
 
 if TYPE_CHECKING:
-    from google.ads.googleads.v14.services.services.customer_service import CustomerServiceClient
-    from google.ads.googleads.v14.services.services.google_ads_service import GoogleAdsServiceClient
-    from google.ads.googleads.v14.services.types.google_ads_service import GoogleAdsRow
+    from google.ads.googleads.v15.services.services.customer_service import CustomerServiceClient
```

Comment:
Isnt this a breaking change?

Reply:
No, we use `CustomerServiceClient`, `GoogleAdsServiceClient` and `GoogleAdsRow` in our code only for type annotation.